client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -0,0 +1,67 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import NoticeOutlineIcon from 'gridicons/dist/notice-outline';
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import BannerNotice from 'components/banner-notice';
+import { reasons } from 'wcpay/disputes/strings';
+import { Dispute } from 'wcpay/types/disputes';
+
+interface DisputeNoticeProps {
+	dispute: Dispute;
+	urgent: boolean;
+}
+
+const DisputeNotice: React.FC< DisputeNoticeProps > = ( {
+	dispute,
+	urgent,
+} ) => {
+	const clientClaim =
+		reasons[ dispute.reason ]?.claim ??
+		__(
+			'The cardholder claims this is an unrecognized charge.',
+			'woocommerce-payments'
+		);
+
+	return (
+		<BannerNotice
+			status={ urgent ? 'error' : 'warning' }
+			icon={ <NoticeOutlineIcon /> }
+			className="dispute-notice"
+			isDismissible={ false }
+		>
+			{ createInterpolateElement(
+				sprintf(
+					/* translators: %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
+					__(
+						// eslint-disable-next-line max-len
+						'<strong>%s</strong> Challenge the dispute if you believe the claim is invalid, or accept to forfeit the funds and pay the dispute fee. Non-response will result in an automatic loss. <a>Learn more about responding to disputes</a>',
```

Comment:
Just to note that this line might change based on feedback from p1692075650439389-slack-C03LA109BKM.